<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>combobox_editor</class>
 <widget class="QMainWindow" name="combobox_editor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="title_label">
      <property name="font">
       <font>
        <family>ONE 모바일POP</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>이름으로 한글, 영어, 띄어쓰기, 숫자만 입력 가능
 중복된 이름 사용 시 저장 불가</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>10</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QTableWidget" name="table">
      <property name="font">
       <font>
        <family>ONE 모바일POP</family>
        <pointsize>13</pointsize>
       </font>
      </property>
      <column>
       <property name="text">
        <string>현재 이름</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>변경 후 이름</string>
       </property>
      </column>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <spacer name="horizontalSpacer_6">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>37</width>
          <height>23</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="up_btn">
        <property name="font">
         <font>
          <family>ONE 모바일POP</family>
          <pointsize>13</pointsize>
         </font>
        </property>
        <property name="text">
         <string>▲</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="down_btn">
        <property name="font">
         <font>
          <family>ONE 모바일POP</family>
          <pointsize>13</pointsize>
         </font>
        </property>
        <property name="text">
         <string>▼</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>23</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="add_btn">
        <property name="font">
         <font>
          <family>ONE 모바일POP</family>
          <pointsize>13</pointsize>
         </font>
        </property>
        <property name="text">
         <string>추가</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="delete_btn">
        <property name="font">
         <font>
          <family>ONE 모바일POP</family>
          <pointsize>13</pointsize>
         </font>
        </property>
        <property name="text">
         <string>삭제</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>37</width>
          <height>23</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>10</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="save_btn">
        <property name="font">
         <font>
          <family>ONE 모바일POP</family>
          <pointsize>13</pointsize>
         </font>
        </property>
        <property name="text">
         <string>저장</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="cancel_btn">
        <property name="font">
         <font>
          <family>ONE 모바일POP</family>
          <pointsize>13</pointsize>
         </font>
        </property>
        <property name="text">
         <string>취소</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
